app/views/base/layout.scala

```diff
@@ -116,6 +116,14 @@ object layout:
       div(id := "dasher_app", cls := "dropdown")
     )
 
+  private def anonDasher(using ctx: PageContext) =
+    val prefs = trans.preferences.preferences.txt()
+    div(cls := "dasher")(
+      a(href := s"${routes.Auth.login.url}?referred=${ctx.req.path}", cls := "signin")(trans.signIn.txt()),
```

Comment:
mispelt `referrer` here, breaking login redirection :open_mouth:

Reply:
oh did you want me to embed those dashers in strings? that was an optimization?

Reply:
prebuilt string html was a silly optimization, I don't mind that they're turned into snabbdom.
the login query parameter is still broken tho